src/employee_personal_dtl_management.c

```diff
@@ -319,6 +319,105 @@ char* raise_grievances(int e_id)
 // Ending of the raise grievance function.
 
 
+// Rate or Feedback the employee
+char* employee_rating()
+{   
+	MYSQL_RES *read=NULL;
+	MYSQL_ROW row=NULL;
+
+	conn8 = mysql_init(NULL);
+	mysql_real_connect(conn8, "localhost", "root", "1234","payroll", 3306, NULL, 0);
+	
+	if(!conn8)
+	{
+		printf("Connection error");
+		return 0;
+	}
+	
+	else
+	{
+		char stmt[1500];
+		char description[200];
+		int e_id=0;
+		int rate=0;
+		int x=0;
+		char *choice;
+		int temp;
+		
+		do
+		{
+			temp = 0;
+			printf("Enter the Employee ID for rating: ");
+  			scanf("%d",&e_id);
+			char qry1[]={"select * from emp_details where emp_id=%d"};
+			sprintf(stmt,qry1,e_id);
+			if (mysql_query(conn8,stmt))
+			{
+        		printf("		Error: %s\n", mysql_error(conn8));
+        		return ("		Failed to execute query.");
+    		}
+    		
+    		else
+    		{
+    			read = mysql_store_result(conn8);
+				row = mysql_fetch_row(read);
+				if(row==NULL)
+				{
+					printf("		Employee ID not found\n");
+					temp=0;
+				}
+				else
+				{
+					temp=1;
+				}
+    		}
+		}while (temp!=1);
+		
+		do
+		{   
+			printf("Enter the rating of the employee: ");
+			scanf("%d",&rate);
```

Comment:
Will it not be better , if we just keep rating as 1,2,3 and 4 . because here user will be able to add any number that's not negative.

Reply:
Yea, thanks, I forget to the validation there.

Reply:
I put the validation, that it will accept from 1 to 5.

Reply:
It Looks good now, Thanks Puneet.